README.md

```diff
@@ -31,20 +31,21 @@
 ## SR-IOV Network Device Plugin
 
 The SR-IOV Network Device Plugin is Kubernetes device plugin for discovering and advertising networking resources in the
-form of SR-IOV virtual functions (VFs) and PCI physical functions (PFs) available on a Kubernetes host.
+form of SR-IOV virtual functions (VFs), Auxiliary network devices (in particular Scalable functions) and PCI physical
+functions (PFs) available on a Kubernetes host.
 
 ## Features
 
-- Handles SR-IOV capable/not-capable devices (NICs and Accelerators alike)
+- Handles SR-IOV capable/not-capable devices (NICs and Accelerators alike, Auxiliry network devices)
```

Comment:
its incorrect spelling.

it should be auxiliary

Reply:
Mistype